src/components/FileMenu/RenameDialog.js

```diff
@@ -117,14 +72,11 @@ export const RenameDialog = ({ type, object, onClose, onRename, onError }) => {
 }
 
 RenameDialog.propTypes = {
-    id: PropTypes.string,
     object: PropTypes.shape({
-        id: PropTypes.string.isRequired,
```

Comment:
I guess this `id` is still always passed but not necessary anymore.

Reply:
True. Will update.